### DISCUSSION TITLE
Route overview and the favorites

### DISCUSSION THREAD

**Discussion by r11d11** [devemux86/cruiser#518] [Q&A] [answered]
When I load a route an then short click on the arrow top left an then  Route overview I get the following result (Cruiser 3.0.43-b18):

![Screenshot_2024-04-28-07-14-28-35_a1](https://github.com/devemux86/cruiser/assets/101214119/28b16d95-c20b-4af7-a970-cb318d98f305)

I am impressed how quick Cruiser shows all my favorites (especially Biergarten from overpaas) but I want to see the route and not all my favorites.

When I click on the route and then ... and then Show Whole route It looks better resp. what I expecteded:
 
![Screenshot_2024-04-28-07-17-40-80_a2](https://github.com/devemux86/cruiser/assets/101214119/7ab08946-7a98-4195-a665-444ce858c824)

When I zoom out with the - button it also looks nice:

![Screenshot_2024-04-28-07-29-57-36_beed72cb86d752c6022398a83c94aee3](https://github.com/devemux86/cruiser/assets/101214119/1e358fcd-24fc-4ff3-9c56-870d84d43d6a)

Is the first picture a bug or a feature?

**Comment by WolfgangZ42** [comment]
Sorry @r11d11 - all of you screenshots are a nightmare for me ... 🥴

My usual navigation screen look like this: 
<img width="400" src="https://github.com/devemux86/cruiser/assets/121806238/7756da36-129c-4d0f-baad-b09466f1d40a"/>

But: everyone as he likes it! 👍

**Reply by r11d11** [reply]
Hi @WolfgangZ42 , maybe everyone should share a screenshot  of how they configured Cruiser in a separate thread . The others will then say that they like their own setup much better. Who would have thought that? But that's another topic.  Unfortunately your answer doesn't help me in any way.

**Comment by devemux86** [comment, marked as answer]
@r11d11 The visibility of favorites depends on the settings we have chosen:

- Favorites screen: we can choose which favorite groups are visible and from which zoom level.

- "Settings | Map | Appearance | Marker groups (zoom)" specifies the zoom where the markers are grouped like image 2 and 3.
Default is zoom 17, have you changed it?

Alternatively, we could introduce a new option in favorites about visibility on / off when navigation is active.

**Reply by MotoUKRider** [reply]
@devemux86 I'm confuzzled... Isn't that what "Use in Navigation" does?

@r11d11 I find a level setting of 10, in the Favorites groups, lets me see items up close but allows me to zoom out and use the map "as a map" without the clutter.
![favorites-import-test](https://github.com/devemux86/cruiser/assets/125696109/f2d577c2-4c2d-457d-b28f-d8ece3ebeedf)

**Reply by devemux86** [reply]
@MotoUKRider the 2 selections are independent:

- "Show on map" turns on / off the favorites visibility on the map with adjustable zoom level

- "Use in navigation" allows the use of favorites in navigation, i.e. display the next nearby 1-4 favorites

**Comment by r11d11** [comment]
> * "Settings | Map | Appearance | Marker groups (zoom)" specifies the zoom where the markers are grouped like image 2 and 3.
>       Default is zoom 17, have you changed it?

Yes I changed it from 17 to 11. Now I put it back to 17.
The many possibilities in Cruiser can be a blessing or a curse.

> I find a level setting of 10, in the Favorites groups, lets me see items up close but allows me to zoom out and use the map "as a map" without the clutter.

I now changed every Favorites group from from 2 to 10. Then the also the Route overview looks much better 👍 

It was a little bit confusing to me, because I had this issue only when I pressed Route overview button
I think I don't need a new option in favorites about visibility on / off when navigation is active.

**Reply by devemux86** [reply]
@r11d11 

"Marker groups (zoom)" 17 means that the markers are grouped in zoom levels < 17.
So if we set a small zoom like 11, then most of the time the markers are not grouped.

**Reply by bios4** [reply]
Funny coincidence - I also was just exploring that setting, see my comment about it [here](https://github.com/devemux86/cruiser/discussions/173#discussioncomment-9251304)... 😀
